apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -1631,4 +1630,513 @@ export const notImplementedCards: DbCard[] = [
+		},
+	},
+	{
+		id: '119',
+		name: '',
+		type: 'field',
+		color: null,
+		description: 'Spells have their attack reduced by 3',
+		effects: {
+			beforeDamage: async function* ({ game }) {
+				for (const combatItem of game.turn.combatStack) {
+					if (combatItem.source?.type === 'spell') {
+						combatItem.value -= 3;
+					}
+				}
+				yield game;
+			},
+		},
+	},
+	{
+		id: '120',
+		name: 'Lucky 7',
+		type: 'spell',
+		colors: ['blue', 'green', 'red'],
+		attack: 10,
+		description: 'Depending on the damage this spell deals, do the following: \
+		- 10 damage: Discard this card. \
+		- 9 damage: Draw 1 card. \
+		- 8 damage: Heal 2 HP. \
+		- 7 damage: Deal another 14 damage to the opponent.',
+		effects: {},
+	},
+	{
+		id: '121',
+		name: 'One one one',
+		type: 'spell',
+		colors: [],
+		attack: 13,
+		description: 'Depending on the damage this spell deals, do the following: \
+		- 13 damage: Heal the opponent for 15HP. \
+		- 12 damage: Discard this card. \
+		- 11 damage: Discard the top card of every opponent stack. \
+		- 10 damage: Your opponent draws 1 card.',
+		effects: {},
+	},
+	{
+		id: '122',
+		name: 'Three two wand: Zero',
+		type: 'spell',
+		colors: [],
+		attack: 3,
+		description: 'Depending on the damage this spell deals, do the following: \
+		- 3 damage: Heal 9 HP. \
+		- 2 damage: Deal another 18 damage to the opponent. \
+		- 1 damage: Discard this card. \
+		- 0 damage: Discard the top field then return this card to your hand.',
+		effects: {},
+	},
+	{
+		id: '123',
+		name: 'Even the Odds',
+		type: 'spell',
+		colors: ['blue'],
+		attack: 15,
+		description: 'If this spell deals even damage, double the damage dealt. If it deals odd damage, discard this card.',
+		effects: {},
+	},
+	{
+		id: '124',
+		name: 'Odd the Evens',
+		type: 'spell',
+		colors: ['red'],
+		attack: 16,
+		description: 'If this spell deals odd damage, heal 7 HP. If it deals even damage, discard this card.',
+		effects: {},
+	},
+	// Archetype based on revealing a card from the opponent's hand and if it's a type or color something happens. I want the theme to be about scrying or looking into the future or looking into a crystal ball
+	{
+		id: '125',
+		name: 'Seer\'s Tent',
+		type: 'field',
+		color: 'blue',
+		description: 'At the start of each turn, each player reveals the top card of their deck then discard it. Depending on the color of the card, do the following: \
+		- Blue: Draw 1 card. \
+		- Red: Deal 5 damage to the opponent. \
+		- Green: Heal 5 HP. \
+		- Colourless: Take 10 damage.',
+		effects: {},
+	},
+	{
+		id: '126',
+		name: 'Crystal Ball',
+		type: 'spell',
+		colors: ['red', 'green', 'blue'],
+		attack: 0,
+		description: 'Draw 1 card and reveal it. Depending on the color of the card, do the following: \
+		- Blue: Deal 17 damage to the opponent. Then discard this card. \
+		- Red: Discard 1 card from your hand and deal 13 damage to the opponent. \
+		- Green: Deal 9 damage to the opponent and heal 9 HP.',
```

Comment:
for multi-line, use backticks and just press enter as per normal, this will make it easier for the UI to just work. I might do a markdown parser so we can do bolds etc, and perhaps some symbols like `{{blue}}` for a special icon